src/config.rs

```diff
@@ -590,12 +617,12 @@ impl Cfg {
         Ok(None)
     }
 
-    fn parse_override_file<S: AsRef<str>>(contents: S) -> Result<OverrideFile> {
+    fn parse_override_file<S: AsRef<str>>(contents: S, toml_only: bool) -> Result<OverrideFile> {
```

Comment:
While booleans are easy, I find myself wishing this was a enum such as:

```rust
pub enum ParseMode {
    OnlyToml,
    Both,
}
```

That way we're calling `Cfg::parse_override_file(contents, ParseMode::Both)` or similar, making it a little clearer.  The compiler will end up reducing it to something as efficient as the boolean, but we're clear at callsites, and type-safe in the face of the API changing in the future.